lib/ansiblelint/color.py

```diff
@@ -0,0 +1,18 @@
+"""Console coloring support."""
+from enum import Enum
+
+
+class C(Enum):
```

Comment:
`C` is a bad name. It doesn't mean anything in code. Is it a constant? A config value? Some magic name?

Reply:
Is it constant-ish and I wanted a short name to ease its re-use in various places. Which name would you like? I do not have strong feelings towards "C".

Reply:
Just `Color`. Because ansible/ansible uses C for configs, for example. Transferring the concept here would be confusing.